java/connector-node/risingwave-source-cdc/src/main/java/io/debezium/connector/mysql/strategy/mysql/MySqlConnection.java

```diff
@@ -0,0 +1,213 @@
+// Copyright 2025 RisingWave Labs
```

Comment:
Which of these jave code are from debezium? 
Which debezium version? RisingWave is using debezium v2.6.
What changes have been made to them to fit RisingWave?

Reply:
refer to https://maven.apache.org/surefire/maven-surefire-plugin/examples/configuring-classpath.html
The Surefire plugin builds the test classpath at runtime in the following order:

1. The [test-classes](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#testClassesDirectory) directory
2. The [classes](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#classesDirectory) directory
3. The project dependencies (with all scopes)
4. Additional classpath elements

Our changes is 2 and debezium source code is 3.